esp/templates/program/modules/studentonsite/schedule.html

```diff
@@ -35,8 +35,8 @@
                 <td rowspan="2" class="time">{{ timeslot.0.short_time }}</td>
                 <td class="cls">
                 {% if cls %}
-                    <!--link to course details?-->
-                    <b>{{ cls.section.title }}{% if not cls.first_meeting_time %} (continued){% endif %}</b>
+                    <!--link to course details-->
+                    <b>{{ cls.section.title }}{% if not cls.first_meeting_time %} (continued){% else %}&nbsp;<a class="no-dec" title="More Details" href="/learn/{{one}}/{{two}}/onsitedetails/{{ cls.section.id }}"><i class="material-icons md-3">list</i></a>{% endif %}</b>
```

Comment:
The little icon is higher than the text for me and I'd be a much happier camper if they were aligned :)
![image](https://user-images.githubusercontent.com/25392075/82273912-badf9300-994c-11ea-82ca-13b64018c1eb.png)

Reply:
Hmm...I could have sworn I made that aligned...

Reply:
Is your class title more than one line long?

Reply:
Nope, just that one word

Reply:
Looks fine to me:
![image](https://user-images.githubusercontent.com/7232514/82350640-3e3acc00-99c1-11ea-9ed0-4cce2748fe75.png)